security/acme-client/src/opnsense/mvc/app/library/OPNsense/AcmeClient/LeValidation/DnsEasydns.php

```diff
@@ -1,7 +1,8 @@
 <?php
 
 /*
- * Copyright (C) 2023 mleinart
+ * Copyright (C) 2023 mleinart,
```

Comment:
Please add a new Copyright line for 2024 with only your name.

Reply:
Done!

Reply:
@fraenki Are there any other changes that need to be made?